<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RealTimeHash</class>
 <widget class="QMainWindow" name="RealTimeHash">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RealTimeHash</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(211, 215, 207);
alternate-background-color: rgb(136, 138, 133);
border-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>hashed output</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="output_field">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Input field</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="input_field">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sha_hash_radio_button">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
</string>
        </property>
        <property name="text">
         <string>Sha512</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="md5_hash_radio_button">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>MD5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="keccak_hash_radio_button">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
selection-background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Keccak_512</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clear_button">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(186, 189, 182);</string>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exit_button">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(186, 189, 182);</string>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>938</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
